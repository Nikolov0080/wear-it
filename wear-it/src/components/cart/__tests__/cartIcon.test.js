import React from 'react';
import renderer from 'react-test-renderer';
import CartIcon from '../cartIcon';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Cart Item', () => {
    const tree = renderer
        .create(<BrowserRouter><CartIcon /></BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});