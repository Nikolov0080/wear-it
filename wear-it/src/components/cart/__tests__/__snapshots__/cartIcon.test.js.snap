// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Cart Item 1`] = `
<div
  className="cartBox"
>
  <a
    href="/cart"
    onClick={[Function]}
  >
    <img
      alt="cart"
      className="cart"
      src="cartImage.png"
    />
  </a>
  <div
    className="cartLabel"
  >
    my cart
  </div>
  <div
    className="cartItems"
  >
    5
  </div>
</div>
`;
